src/libstore/remote-store.cc

```diff
@@ -39,12 +89,12 @@ RemoteStore::RemoteStore(const Params & params)
                 }
                 return conn;
             },
-            [this](const ref<Connection> & r) {
+            [config](const ref<Connection> & r) {
```

Comment:
With a `this` reference, we don't have to think about the lifetime as much.

Reply:
Sure. I will try not to think about the double indirection :).